.github/actions/cache/action.yml

```diff
@@ -4,27 +4,42 @@ inputs:
   use-rust:
     description: 'Whether rust is used'
     default: false
+  extra-key:
+    description: 'Extra cache key to use in the cache name. Useful when several caches are used in one workflow.'
```

Comment:
Nit: In the second sentence "Useful when ...", we could mention this is true when matrix is used.